src/main/resources/python/components/schemas/schema_cls/_types.hbs

```diff
@@ -20,6 +20,7 @@ types: typing.FrozenSet[typing.Type] = frozenset({
 {{/eq}}
 {{#eq this "integer"}}
     int,
+    float,
```

Comment:
int + float should be allowed in for int types, and the values will be checked to ensure that they are integer like.
In the future a flag could be added floatingTypesAllowedForTypeInteger to disallow float here in python and Float/Double in java